packages/core/src/api/measurements/initialize.test.ts

```diff
@@ -236,4 +247,54 @@ describe('api.measurements', () => {
       });
     });
   });
+
+  describe('User action', () => {
+    it('buffers the error if a user action is in progress', () => {
```

Comment:
same here, should this just say 

```suggestion
    it('buffers the item if a user action is in progress', () => {
```
?